dockerplugin.py

```diff
@@ -659,16 +763,73 @@ def debug(self, msg):
             print 'DEBUG: ', msg
 
     collectd = ExecCollectd()
+
+    # importing here because collectd must be instantiated first in order to
+    # log the import error
+    try:
+        import argparse
+    except ImportError as e:
+        log.exception(("Unable to import the library 'argparse'. "
+                       "Please install argparse using pip"))
```

Comment:
If argparse is not installed we should fail the plugin run. The exception message should be descriptive enough such that we dont need to log our own error.